packages/fast-form-generator-react/src/form/form-category.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import IFormItemCommon from "./form-item";
+import styles from "./form-category.style";
+import { IFormCategoryClassNameContract } from "../class-name-contracts/";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * Select state interface
+ */
+export interface IFormCategoryProps {
+
+    /**
+     * Passes the category item
+     */
+    categoryItem: JSX.Element[];
+
+    /**
+     * Passes the category title
+     */
+    title: string;
+
+    /**
+     * Is the category expandable
+     */
+    Expandable?: boolean;
+
+    /**
+     * Is the category expanded
+     */
+    isExpanded?: boolean;
+
+    /**
+     * Passes the id

```

Comment:
Can you make the id comment a bit more descriptive? - the prop isn't actually doing any passing itself